src/Assignment6/Bankingsystem/Inputvalidators.cs

```diff
@@ -0,0 +1,83 @@
+﻿// <copyright file="Inputvalidators.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace Bankingsystem
+{
+    using System.Text.RegularExpressions;
+
+    /// <summary>
+    /// Has methods to validate inputs.
+    /// </summary>
+    internal static class Inputvalidators
+    {
+        /// <summary>
+        /// Gets String as input and converts it into integer.
+        /// </summary>
+        /// <returns>Integer</returns>
+        public static int IntegerValidator()
+        {
+            int integer;
+            string? n = Console.ReadLine();
+            while (!int.TryParse(n, out integer))
+            {
+                Console.WriteLine("Enter valid Number");
+                n = Console.ReadLine();
+            }
+
+            return integer;
+        }
+
+        /// <summary>
+        /// Gets String as input and validates it.
+        /// </summary>
+        /// <returns>Non empty string</returns>
+        public static string StringValidator()
+        {
+            string? n = Console.ReadLine();
```

Comment:
use a proper variable name instead of a single character name.

Reply:
Variable name `n `is changed to `inputRead`.